sample_datasets/snips-to-rasa.py

```diff
@@ -0,0 +1,67 @@
+import json
+
+inputFiles = [
+    '2017-06-custom-intent-engines/GetWeather/train_GetWeather_full.json',
+    '2017-06-custom-intent-engines/AddToPlaylist/train_AddToPlaylist_full.json',
+    '2017-06-custom-intent-engines/BookRestaurant/train_BookRestaurant_full.json',
+    '2017-06-custom-intent-engines/GetWeather/train_GetWeather_full.json',
+    '2017-06-custom-intent-engines/PlayMusic/train_PlayMusic_full.json',
+    '2017-06-custom-intent-engines/RateBook/train_RateBook_full.json',
+    '2017-06-custom-intent-engines/SearchCreativeWork/train_SearchCreativeWork_full.json',
+    '2017-06-custom-intent-engines/SearchScreeningEvent/train_SearchScreeningEvent_full.json',
+]
+
+for inputFile in inputFiles:
+    print('Now converting: ' + inputFile)

```

Comment:
please use a logge, so at the beginning of the file 

```
import logging

logger = logging.getLogger(__name__)
```

and then `logger.debug(...)`